mne/viz/_3d.py

```diff
@@ -1363,6 +1378,126 @@ def plot_dipole_locations(dipoles, trans, subject, subjects_dir=None,
     return fig
 
 
+def plot_dipole_locations(dipoles, trans, subject, subjects_dir=None,
+                          bgcolor=(1, 1, 1), opacity=0.3,
+                          brain_color=(1, 1, 0), fig_name=None,
+                          fig_size=(600, 600), mode=None,
+                          scale_factor=0.1e-1, colors=None,
+                          coord_frame='head', idx='gof',
+                          show_all=True, ax=None, block=False,
+                          show=True, verbose=None):
+    """Plot dipole locations
+
+    If mode is set to 'cone' or 'sphere', only the location of the first
+    time point of each dipole is shown else use the show_all parameter.
+
+    The option mode='orthoview' was added in version 0.14.
+
+    Parameters
+    ----------
+    dipoles : list of instances of Dipole | Dipole
+        The dipoles to plot.
+    trans : dict
+        The mri to head trans.
+    subject : str
+        The subject name corresponding to FreeSurfer environment
+        variable SUBJECT.
+    subjects_dir : None | str
+        The path to the freesurfer subjects reconstructions.
+        It corresponds to Freesurfer environment variable SUBJECTS_DIR.
+        The default is None.
+    bgcolor : tuple of length 3
+        Background color in 3D.
+    opacity : float in [0, 1]
+        Opacity of brain mesh.
+    brain_color : tuple of length 3
+        Brain color.
+    fig_name : str
+        Mayavi figure name.
+    fig_size : tuple of length 2
+        Mayavi figure size.
+    mode : str
+        Should be ``'cone'`` or ``'sphere'`` or ``'orthoview'`` to specify
+        how the dipoles should be shown. If orthoview then matplotlib is
+        used otherwise it is mayavi.
```

Comment:
once we remove the mode param as only orthoview is valid we'll add a backend param or something